src/SignalR/clients/ts/signalr/src/HubConnectionBuilder.ts

```diff
@@ -13,6 +13,31 @@ import { JsonHubProtocol } from "./JsonHubProtocol";
 import { NullLogger } from "./Loggers";
 import { Arg, ConsoleLogger } from "./Utils";
 
+// tslint:disable:object-literal-sort-keys
+const LogLevelNameMapping = {
```

Comment:
Does this need to be exported now that it's included in the public configureLogging type signature?

It might be better to change configureLogging to take a `keyof typeof LogLevel` `keyof typeof LogLevelNameMapping`. You can still use the LogLevelNameMapping and call toLowerCase on the argument.

As an upside, hat would be make things more consistent with the capitalization of our HubConnectionStates.

Reply:
`keyof typeof LogLevel` doesn't work since we want to allow "warn" and "info". I say don't export anything new, and change the configureLogging type signature to take a string with a link to our docs from the doc comment.